Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -154,12 +198,83 @@ public extension Array where Element: Equatable {
     /// - Returns: an array of unique elements.
     ///
     func withoutDuplicates() -> [Element] {
-        // Thanks to https://github.com/sairamkotha for improving the method
```

Comment:
So here that would be applicable as optimization
```swift
func withoutDuplicates() -> [Element] where Element: Hashable {
    var set: Set<Element> = []
    return filter { set.insert($0).inserted }
}
```

Reply:
Thank you @LucianoPAlmeida. Added your version to `public extension Array where Element: Hashable { }`.